src/components/InputField/validation.js

```diff
@@ -1,7 +1,8 @@
 const defaultValidationPatterns = {
   tel: /^[0-9 ]{11,}$/,
   number: /^[0-9]+$/,
-  email: /^([A-Za-z0-9_+-]+\.?)*[A-Za-z0-9_+-]+@[A-Za-z0-9]+([A-Za-z0-9_-]+\.?)*[A-Za-z0-9]+\.[A-Za-z]{2,}$/,
+  // eslint-disable-next-line no-useless-escape
```

Comment:
Were you not able to get around this? Not a big deal, but we're gonna have to use this pattern in multiple projects; would be nice to get it clean

Reply:
Oops forgot this
Done

Reply:
Thanks man, will doublecheck once the test env has rebuilt